cmd/collector/app/flags/flags.go

```diff
@@ -0,0 +1,255 @@
+// Copyright (c) 2019 The Jaeger Authors.
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flags
```

Comment:
this was a move from app/flags.go, but unfortunately git does not recognize it as move due to many changes.

Reply:
This commit shows the actual diffs https://github.com/jaegertracing/jaeger/pull/3710/commits/cb6a8c9e7d25f4a92a428dd7b56ec71a43675509